singularity/code/savegame.py

```diff
@@ -693,34 +693,35 @@ def savegame_exists(savegame_name):
 
     return True
 
-def sanitize_filename(filename):
-    """Create filename that's safe for all operating systems"""
+def check_filename_illegal(filename):
```

Comment:
At the moment, you can put in an empty / whitespace-only filename, hit OK and get no error (but no savegame either).  Could you please add a case for this while we are at it? :)

I am thinking something like

```python

    if not filename.strip():
        return _('Filename must not be empty nor entirely made of whitespace')
```

It is hardly a new problem, but it would be nice to fix it along with all the other checking. :)

Reply:
Thanks for catching this one, will do :)